docs/src/pages/company/careers/engineering-manager.md

```diff
@@ -0,0 +1,74 @@
+# Developer Advocate
```

Comment:
We should probably drop this file, it is not updated, and we are not opening this role right now.